gnocchi/rest/aggregates/processor.py

```diff
@@ -156,21 +156,22 @@ def aggregated(refs_and_timeseries, operations, from_timestamp=None,
                     'timespan (%.2f%%)' % (needed_percent_of_overlap,
                                            percent_of_overlap))
 
-        granularity, times, values, is_aggregated = (
+        granularity, times, values = (
             agg_operations.evaluate(operations, key, times, values,
-                                    False, references[key]))
+                                    references[key]))
 
-        if is_aggregated:
+        values = values.T
+        if len(values) == 1 and len(references[key]) > 1:
```

Comment:
If I just do: "(* 2 (metric id1 mean))" that doesn't aggregated anything, but with your change I got "aggregated": .... instead of "id1_mean" as key.

Reply:
I have added some tests to cover this: https://github.com/gnocchixyz/gnocchi/pull/442

Reply:
no, it'll give id1_mean. it'll only give "aggregated" if the number of original metrics was greater than 1 and shrinks to 1. i know this because you added tests for this already. :)

this case doesn't cover a scenario where you only aggregate a few of the series...  that said, i don't think the code supports that.

Reply:
ah, nevermind. i see your new test for reaggregate a single series returns aggregated